nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedRecipe.groovy

```diff
@@ -132,11 +134,6 @@ class HelmHostedRecipe
         .handler(hostedHandlers.delete)
         .create())
 
-    builder.route(new Route.Builder()
-        .matcher(BrowseUnsupportedHandler.MATCHER)
```

Comment:
minor: the `BrowseUnsupportedHandler` import at the top of this file is now unused and can be removed